build.rs

```diff
@@ -0,0 +1,59 @@
+use std::env;
+use std::fs;
+use std::process::Command;
+
+fn main() {
+  let mut curr_dir = env::current_dir().expect("Get project directory");
+  curr_dir.push("src");
+  curr_dir.push("parquet_thrift");
+
+  // Check if thrift is installed
+  if let Err(error) = run(Command::new("which").arg("thrift")) {
```

Comment:
We may also need to check version number, such that the major version should be greater than 1 or that the minor version should be greater than 10. You can also just add a TODO here.

Reply:
Yes, will do. Thanks.